integration_tests/backend/test_backend.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import requests
+import subprocess
+from pathlib import Path
+import os
+import sys
+
+import aqueduct
+
+
+def get_response(endpoint, additional_headers={}):
+    headers = {"api-key": pytest.api_key}
+    headers.update(additional_headers)
+    url = f"{pytest.server_address}{endpoint}"
+    if url[:4] != "http":
+        try:
+           r = requests.get("https://"+url, headers=headers) 
+        except:
+            try:
+                r = requests.get("http://"+url, headers=headers) 
+            except:
+                raise Exception(f"Cannot connect to {url}")
+    else:
+        try:
+            r = requests.get(url, headers=headers)
+        except:
+            raise Exception(f"Cannot connect to {url}")
+    return r
+
+class TestBackend:
+    GET_WORKFLOW_TABLES_TEMPLATE = "/api/workflow/%s/tables"
+    WORKFLOW_PATH = Path(__file__).parent / "setup"
```

Comment:
wait I've never seen this before, how does the "/" not error here? The right side isn't wrapped in a string.

Reply:
ok sg lets just leave it this way